token/program-2022/src/extension/mod.rs

```diff
@@ -1124,12 +1125,51 @@ impl Extension for AccountPaddingTest {
     const TYPE: ExtensionType = ExtensionType::AccountPaddingTest;
 }
 
+/// Packs an unsized extension into a new TLV space
```

Comment:
Actually this just packs arbitrary bytes. It's not clear what would fail if the bytes were incompatible with unsized extension type V.

Reply:
That's fair, will update